incubator/hnc/Makefile

```diff
@@ -52,37 +52,44 @@ test: build
 # Builds all binaries (manager and kubectl) and manifests
 build: generate fmt vet manifests
 	go build -o bin/manager ./cmd/manager/main.go
-	go build -o bin/kubectl/kubectl-hnc ./cmd/kubectl/main.go
+	go build -o bin/kubectl/kubectl-hierarchical_namespaces ./cmd/kubectl/main.go
 
 # Clean all binaries (manager and kubectl)
-clean:
-	rm -r bin/kubectl/kubectl-hnc
-	rm -f bin/manager
-	
+clean: krew-uninstall
+	-rm -rf bin/*
+	-rm -rf manifests/*
+	-rm -f ${GOPATH}/bin/kubectl-hierarchical_namespaces
+	-rm -f ${GOPATH}/bin/kubectl-hns
+
 # Install kubectl plugin
 kubectl: build
-	cp bin/kubectl/kubectl-hnc ${GOPATH}/bin/kubectl-hnc
-	@echo "Installed kubectl-hnc to GOPATH/bin"
+	cp bin/kubectl/kubectl-hierarchical_namespaces ${GOPATH}/bin/kubectl-hierarchical_namespaces
```

Comment:
Would we want to delete this and the sym link when uninstalling via krew?

Reply:
I don't _think_ so because `make krew-install` doesn't depend on `make kubectl` so these typically won't be here. `make clean` deletes them both.

What would be the rationale for adding that here? "Just in case?"

Reply:
Yeah, just more or less asking the question to see if it makes sense.